apps/jps/preview/base/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ resources:
   - ../../../base   # This is apps/base instead of apps/jps/base as preview doesn't install everything
   - ../../identity/identity.yaml
   - ../../jps-judicial-payment-service/jps-judicial-payment-service.yaml
+  - ../../jps-judicial-payment-frontend/jps-judicial-payment-frontend.yaml
```

Comment:
Why do u need this in preview? Deployments to preview are managed through Jenkins. Please remove both the apps in here.